python/sglang/srt/layers/quantization/fp8_kernel.py

```diff
@@ -95,9 +98,13 @@ def per_token_group_quant_fp8(
     assert x.is_contiguous(), "`x` is not contiguous"
 
     finfo = torch.finfo(dtype)
-    fp8_min = finfo.min
     fp8_max = finfo.max
 
+    if is_hip_:
+        fp8_max = 224.0
+
+    fp8_min = -fp8_max
```

Comment:
only do this for `is_hip_`

Reply:
fp8_min is always negative of fp8_max:
On AMD: fp8_min = -224.0, fp8_max = 224.0
On Nvidia: fp8_min = -448.0, fp8_max = 448.0